pom.xml

```diff
@@ -8,11 +8,15 @@
     <description>SQLite Knowledge Base Connector for Rosette Server</description>
     <properties>
         <rosapi.home>/must-be-specified</rosapi.home> <!-- where Rosette Server is installed -->
-        <rosapi-version>1.17.0</rosapi-version>
-        <rex-je-version>7.46.1.c62.2</rex-je-version>
-        <rex-osgi-version>7.46.1.c62_2</rex-osgi-version>
-        <flinx-version>0.4.26.c62.2</flinx-version>
+        <rosapi-version>1.24.0</rosapi-version>
+        <rex-je-version>7.55.0.c68.0</rex-je-version>
+        <rex-osgi-version>7.55.0.c68_0</rex-osgi-version>
```

Comment:
Yes, that is correct.  For some reason all the osgi jars have an underscore there instead of a dot, if you take a look at the bundles in Rosette Server.